src/FirebirdSql.Data.FirebirdClient/FirebirdClient/FbCommand.cs

```diff
@@ -198,6 +198,16 @@ public int FetchSize
 		}
 	}
 
+	/// <summary>
+	/// Gets collection of parameters parsed from the query text by <see cref="NamedParametersParser"/> during command prepare.
+	/// </summary>
+	[Category("Data")]
+	[DefaultValue(new string[] { })]
```

Comment:
Not needed for getter-only properties. In fact it should be maybe hidden for designers.